models/marts/dim_customers.sql

```diff
@@ -24,7 +24,10 @@ customer_orders as (
     group by 1
 
 ),
-
+fact_orders as

```

Comment:
To keep consistent nomenclature, I recommend naming the CTE the same as the model you are referencing (like `fct_orders`) or just merely calling this the `orders` CTE.